src/emoji.js

```diff
@@ -3,7 +3,11 @@
 
 const search = require('./search')
 const { getRandomSkinTone } = require('./utils')
-const pasteByDefault = getenv('snippetapp')
+const returnAction = getenv('return_action')
+const pasteByDefault = getenv('snippetapp') || (
+  // Only use returnAction if snippetapp is not set
+  !getenv('snippetapp') && returnAction === 'paste'
+)
```

Comment:
I believe we can simplify this:
```suggestion
const pasteByDefault = getenv('snippetapp') ?? returnAction === 'paste'
```

Consider the following script:

```js
ObjC.import('stdlib')

let foo
try {
  foo = $.getenv('foo')
} catch {}

console.log(foo ?? 'not set')
```

Then:

1. `osascript -l JavaScript index.js` outputs "not set"
2. `foo=foo osascript -l JavaScript index.js` outputs "foo"

Thus, if `snippetapp` is not set `getenv('snippetapp')` will result in `null`. So the nullish coalescing operator (`??`) will then return the value of `returnAction === 'paste'`.